ts/api/backend.ts

```diff
@@ -347,9 +350,22 @@ export function BackendClient(
     postAttivaRpt: withBearerToken(
       createFetchRequestForApi(attivaRptT, options)
     ),
-    getPaymentId: withBearerToken(
-      createFetchRequestForApi(getPaymentIdT, options)
-    ),
+    getPaymentId: (maxPollingRetries: number, retryDelay: Millisecond) => {
```

Comment:
`maxPollingRetries` and `retryDelay` don't change between calls to `getPaymentId`, you can move these options to the `BackendClient` constructor